src/bundle/Resources/public/js/OnlineEditor/toolbars/config/base-table.js

```diff
@@ -1,21 +1,12 @@
 import AlloyEditor from 'alloyeditor';
+import EzConfigButtonsBase from './base-buttons';
 
-export default class EzConfigTableBase {
+export default class EzConfigTableBase extends EzConfigButtonsBase {
     constructor(config) {
-        this.name = this.getConfigName();
-
-        const editAttributesButton = config.attributes[this.name] || config.classes[this.name] ? `${this.name}edit` : '';
+        super(config);
 
-        this.buttons = [
-            'ezmoveup',
-            'ezmovedown',
-            editAttributesButton,
-            'tableHeading',
-            'ezembedinline',
-            'ezanchor',
-            'eztableremove',
-            ...config.extraButtons[this.name],
-        ];
+        this.name = this.getConfigName();
```

Comment:
Are these two needed if you're calling `super(config)` already?

Reply:
But the parent doesn't have this assigned.